apps/frontend/src/__fixtures__/algolia.ts

```diff
@@ -1,20 +1,10 @@
-import {
-  ResearchOutputSearchResponse,
-  ResearchOutputSearchResponseEntity,
-  SearchResponse,
-  SearchResponseEntityMetadata,
-} from '@asap-hub/algolia';
 import { createResearchOutputResponse } from '@asap-hub/fixtures';
-import { ResearchOutputResponse } from '@asap-hub/model';
+import { SearchResponse, EntityRecord } from '@asap-hub/algolia';
 
-export const createAlgoliaResponse = <
-  TEntity extends Record<string, unknown>,
-  TEntityName extends string,
->(
-  data: TEntity[],
-  overrides: Partial<SearchResponse<TEntity, TEntityName>> = {},
-  type: string = 'research-output',
-): SearchResponse<TEntity, TEntityName> => ({
+export const createAlgoliaResponse = (
+  data: EntityRecord<'research-output'>[],
+  overrides: Partial<SearchResponse<EntityRecord<'research-output'>>> = {},
+): SearchResponse<EntityRecord<'research-output'>> => ({
```

Comment:
'research-output' could be a constant or part of an enumeration?

Seems like it's from: 

```
export type EntityResponses = {
  'research-output': ResearchOutputResponse;
  user: UserResponse;
};```